src/admin-views/editor/list-row.php

```diff
@@ -115,3 +117,19 @@ class="tribe-ticket-field-order"
 		?>
 	</td>
 </tr>
+<?php if ( $needs_warning ) : ?>
+		<tr class="warning_row">
```

Comment:
Does this work if you have more than one ticket and you try to drag and drop tickets around with ET+?

Reply:
Oh, good catch - probably not. OK, time to re-strategize!